cmd/pop/cli/start.go

```diff
@@ -29,12 +29,15 @@ type PopConfig struct {
 	regions      string
 	replInterval time.Duration
 	// Exported fields can be set by survey.Ask
-	Bootstrap    string `json:"bootstrap"`
-	Capacity     string `json:"capacity"`
-	MaxPPB       int    `json:"maxppb"`
-	FilEndpoint  string `json:"fil-endpoint"`
-	FilToken     string `json:"fil-token"`
-	FilTokenType string `json:"fil-token-type"`
+	Bootstrap           string `json:"bootstrap"`
+	Capacity            string `json:"capacity"`
+	MaxPPB              int    `json:"maxppb"`
+	FilEndpoint         string `json:"fil-endpoint"`
+	FilToken            string `json:"fil-token"`
+	FilTokenType        string `json:"fil-token-type"`
+	ProviderDomainToken string `json:"provider-domain-token"`
+	ProviderDomainName  string `json:"provider-domain-name"`
+	ProviderSubdomain   string `json:"provider-subdomain"`
```

Comment:
Not sure about the word "provider", we need a word that wraps provider & facilitator